docs/source/tutorials/token_classification/entity_recognition_training.ipynb

```diff
@@ -0,0 +1,737 @@
+      ".               0\tO         \n"
+     ]
+    }
+   ],
+   "source": [
+    "i = 0 \n",
+    "\n",
+    "print('Word\\t\\tLabel\\tEntity\\n-------------------------------') \n",
+    "for word, label in zip(given_words[i], given_labels[i]): \n",
+    "    print('{:14s}{:3d}\\t{:10s}'.format(word, label, entities[label])) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "16f8639b",
+   "metadata": {},
+   "source": [
+    "Next, obtain the sentences with some minor pre-processing for readability. Sentences that contain `#` or have length shorter than or equal to one character are removed. The first condition is to ensure that the symbol does not get confused with the same symbol representing subwords after the sentence is being tokenized. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "8f19eeae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sentences = list(map(get_sentence, given_words)) \n",
+    "\n",
+    "sentences, mask = filter_sentence(sentences) \n",
+    "given_words = [words for m, words in zip(mask, given_words) if m] \n",
+    "given_labels = [labels for m, labels in zip(mask, given_labels) if m] "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "50c496b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Number of sentences: 20718\n",
+      "Eu rejects German call to boycott British lamb.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print('Number of sentences: %d' % len(sentences)) \n",
+    "print(sentences[0]) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0fb1b2b0",
+   "metadata": {},
+   "source": [
+    "## 3. Train Models using Cross-Validation \n",
+    "\n",
+    "To identify potential label errors in the training dataset, we compute the out-of-sample predicted probabilities using cross-validation. We first partition the dataset into k-folds: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "797df997",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'folds/' already exists, skipping...\n"
+     ]
+    }
+   ],
+   "source": [
+    "lines = [[]] \n",
+    "for filepath in filepaths: \n",
+    "    for line in open(filepath) : \n",
+    "        if len(line) == 0 or line.startswith('-DOCSTART') or line[0] == '\\n':\n",
+    "            if len(lines[-1]) > 0: \n",
+    "                lines.append([]) \n",
+    "        else: \n",
+    "            lines[-1].append(line) \n",
+    "        \n",
+    "lines = lines[:-1] \n",
+    "lines = [line for m, line in zip(mask, lines) if m] \n",
+    "\n",
+    "k = 5 \n",
+    "indicies = create_folds(lines, k=k) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "304653aa",
+   "metadata": {},
+   "source": [
+    "We train one model for each training/testing pair: "
```

Comment:
"for each training/testing pair"
->
"for each fold's training/testing pair"

Reply:
Add Warning statement that this block will take a long time to execute. 
Also add statement that this block should be run on GPU, otherwise it will take forever.